cmd/eksctl-anywhere/cmd/get.go

```diff
@@ -51,7 +51,8 @@ func getResources(ctx context.Context, resourceType string, output string, args
 	if output != "" {
 		params = append(params, executables.WithOutput(output))
 	}
-	packages, err := kubectl.GetResources(ctx, resourceType, params...)
+	stdOut, err := kubectl.GetResources(ctx, resourceType, params...)
+	packages := stdOut.String()
 	if err != nil {
 		fmt.Print(packages)
```

Comment:
It is actually not a debug item. In case some packages are successful, and other packages fail, kubectl prints the successful packages then prints the packages that failed as an error.